region/brinstar/blue.json

```diff
@@ -413,6 +374,19 @@
                     "For Screw, just entering with a spin jump would work."
                   ]
                 },
+                {
+                  "name": "Spark into Room",
+                  "notable": false,
+                  "entranceCondition": {
+                    "comeInWithSpark": {}
+                  },
+                  "requires": [
+                    {"shinespark": {
+                      "frames": 40,
+                      "excessFrames": 9
+                    }}
+                  ]
```

Comment:
This kills all of the hoppers unless you get a normalized RNG desync, and enter at the top of the door.

Reply:
Thanks, forgot to copy over that `"clearsObstacles": ["C"]`. Did that now.